packages/edit-site/src/components/app/index.js

```diff
@@ -55,6 +62,7 @@ export default function App() {
 			routes={ routes }
 			pathArg="p"
 			beforeNavigate={ beforeNavigate }
+			context={ { currentTheme, themeSupports } }
```

Comment:
To go along with my suggestion to rename context and spread this value when it’s passed to a area resolver function:
```suggestion
			matchResolverArgs={ { siteData: { currentTheme, themeSupports } } }
```

Also, I thought it’d be best to memoize the value passed here so the memo hook in `useMatch` wouldn’t run more often. Yet as things are, this seems fine. The new values from `useSelect` aren’t going to cause `App` to rerender more often and the value here is practically "memoized".